modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogZoneDescriptorSerializers.java

```diff
@@ -149,6 +153,7 @@ public void writeTo(CatalogZoneDescriptor descriptor, CatalogObjectDataOutput ou
 
             output.writeVarInt(descriptor.partitions());
             output.writeVarInt(descriptor.replicas());
+            output.writeVarInt(descriptor.quorumSize());
```

Comment:
I will leave this comment as a placeholder; we need to find out whether we still can modify these serializers with consideration of nearest release